tests/storage/storage_migration/conftest.py

```diff
@@ -139,10 +139,15 @@ def target_storage_class(request, cluster_storage_classes_names):
 def vm_for_storage_class_migration_with_instance_type(
     unprivileged_client,
     namespace,
-    golden_images_fedora_data_source,
+    golden_images_namespace,
     source_storage_class,
     cpu_for_migration,
 ):
+    golden_images_fedora_data_source = DataSource(
+        namespace=golden_images_namespace.name,
```

Comment:
client?

Reply:
client should always be passed, thanks @geetikakay